modules/meb_api/spec/serializers/submission_serializer_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe SubmissionSerializer, type: :serializer do
+  subject { serialize(object, serializer_class: described_class) }
+
+  let(:object) { OpenStruct.new(education_benefit: { '@type': 'Chapter33' }) }
+
+  let(:data) { JSON.parse(subject)['data'] }
+  let(:attributes) { data['attributes'] }
+
+  it 'returns a blank id' do
+    expect(data['id']).to eq('')
+  end
+
+  it 'serializes the education_benefit attribute' do
+    expect(attributes['education_benefit']).to eq({ '@type' => 'Chapter33' })
+  end
+end
```

Comment:
Please add a new line to the end of all new files. If you use a code editor, you can configure your new files to always include a new line at the end.